examples/django_app/requirements.txt

```diff
@@ -1,2 +1,4 @@
 django>=1.8.0,<2.0.0
 chatterbot>=0.5.0
+gunicorn
+whitenoise
```

Comment:
Was there a particular reason gunicorn and whitenoise are needed in the _example_ app?